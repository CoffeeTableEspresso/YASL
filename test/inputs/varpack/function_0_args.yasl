
fn f() {
    return 1, 2
}

fn g() {
    return .a, .b, .c, .d
}

fn h() {
    return 'first', 'second', 'third'
}

let a, let b, let c = f()

echo a
echo b
echo c

a, b, c = g()

echo a
echo b
echo c

a, b, c = h()

echo a
echo b
echo c
