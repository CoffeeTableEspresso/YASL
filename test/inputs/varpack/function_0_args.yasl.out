1
2
undef
a
b
c
first
second
third
